pkg/agent/features/feature.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2024 The Volcano Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package features
+
+type Feature string
+
+const (
```

Comment:
Is the feature switch enabled or disabled by default? How to set the feature switch?

Reply:
All features are enabled by default, and it can be disabled by set param flag `--supported-features`.